e2e/actors/base.js

```diff
@@ -223,7 +234,9 @@ module.exports = {
   },
 
   async navigateToCaseDetailsAs(user, caseId) {
-    await I.goToPage(config.baseUrl, user);
```

Comment:
please change this method to following: 
 await I.goToPage(config.baseUrl, user); 
 
  await this.navigateToCaseDetails(caseId);
 
 and delete all other lines